script/testing/reporting/report_result.py

```diff
@@ -50,37 +51,78 @@ def report_microbenchmark_result(env, timestamp, config,
         'parameters': parameters,
         'metrics': metrics
     }
-    send_result(env, '/microbenchmark/', config.publish_results_username,
+    _send_result(env, '/microbenchmark/', config.publish_results_username,
                 config.publish_results_password, result)
 
 
 def report_artifact_stats_result(env, metrics, username, password):
+    """
+    Parse and format the data from the artifact stats into a JSON body,
+    which is then sent to the performance storage service.
+
+    Parameters
+    ----------
+    env : string
+        The type of environment that we are running in.
+        Valid options are the keys to PERFORMANCE_STORAGE_SERVICE_API.
+    metrics : ???
+        what this
```

Comment:
Its a dict of all the artifact stats gathered.

Reply:
added, thanks!